docs/guide/android-dev-env.md

```diff
@@ -104,6 +106,7 @@ While it’s possible to do this using Android Studio, we’ll focus on the comm
    > - With `;android-28;`, we assumed SDK 28 here, but other APIs are supported just the same.
    > - The `;default;` part replaces `;google_apis;`, which is the default, and is what matters here.
    > - If you are running on a M1 you must install an arm64 architecture i.e. `system-images;android-28;default;arm64-v8a`
+   > - To get a list of available images, run `$ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --list`
```

Comment:
I'm not at all confident about having `cmdline-tools` so deterministically set up in all environments (my doesn't have it...). I think the safer / alternate path to point out is `$ANDROID_HOME/tools/bin/sdkmanager`.